playbooks/robusta_playbooks/event_enrichments.py

```diff
@@ -272,6 +272,34 @@ def pod_events_enricher(event: PodEvent, params: EventEnricherParams):
         )
 
 
+@action
+def enrich_pod_with_node_events(event: PodEvent, params: EventEnricherParams):
+    """
+    Given a Kubernetes pod, fetch related events in the near past for its node
+    """
+    pod = event.get_pod()
+    node: Node = Node.readNode(pod.spec.nodeName).obj
```

Comment:
Can you do a None test on the pod and also use the pod get_node function you added here?